src/environments/environment.ts

```diff
@@ -3,7 +3,12 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  hostname: 'identity.bitclout.com',
```

Comment:
perhaps name this `identityHostname`? please use camel case

Reply:
Added the proper commit.